RTN-041.tex

```diff
@@ -36,17 +46,22 @@
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-DM delivered software to Operations to perform a DRP-like processing of the DESC DC2 data, enable image services in the RSP API Aspect, and provide enhancements to the RSP Portal Aspect and Qserv for interaction with the results.
-The release of this was called Data Preview 0.2 and the production of the data products and publication of them were carried out in an operational manner.
-This provided valuable insights for operational data releases.
+DM delivered software to Operations to perform a DRP-like processing of the DESC DC2 data, enable image services in the RSP API Aspect, and provide enhancements to the RSP Portal Aspect and Qserv for interaction with the results. 
+Detailed quality assessment was carried out on the resultant data products by the System Performance Verification and Validation team. 
+In June 2022, these data products and services were released to a community of 600 delegates as Data Preview 0.2 (DP0.2), accompanied by detailed documentation and tutorials provided by the System Performance Community Science team. 
+The Community Science team continues to support DP0.2 delegates to do science with the DP0.2 dataset via the Rubin Community Forum and a series of regular virtual "Delegate Assemblies" at which members of the community can present science enabled by the DP0.2 dataset. 
+The entire process from the initial ingestion of the DESC DC2 simulated images through processing, QA, release, documentation and ongoing support for science by Rubin Community was carried out in an operational manner and has provided valuable insights for eventual LSST operational data releases.
+The large and varied number of science analyses contributed by the DP0.2 delegates is testament to the success of Data Preview 0. 
 }
 
+
 % Change history defined here.
 % Order: oldest first.
 % Fields: VERSION, DATE, DESCRIPTION, OWNER NAME.
 % See LPM-51 for version number policy.
 \setDocChangeRecord{%
   \addtohist{1}{2022-08-02}{Unreleased.}{William O'Mullane}
+  \addtohist{2}{2023-04-05}{Added System Performance sections}{Leanne Guy}
```

Comment:
ISO dates  .. alos this should now problay be 1.0 and draft could be removed.
```suggestion 
  \addtohist{1.0}{2023-05-04}{Added System Performance sections}{Leanne Guy}
```

Reply:
It is ISO - just the wrong ISO date ..\